azurerm/data_source_policy_definition.go

```diff
@@ -0,0 +1,80 @@
+package azurerm
+
+import (
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2018-05-01/policy"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceArmPolicyDefinition() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceArmPolicyDefinitionRead,
+		Schema: map[string]*schema.Schema{
+			"display_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
```

Comment:
Could we validate this with `validate.NoZerovalues`?

Reply:
I went with `validate.NoEmptyStrings` instead -- cool?